scripts/get-data-local.sh

```diff
@@ -1,3 +1,8 @@
+# Abort on error: https://bertvv.github.io/cheat-sheets/Bash.html#writing-robust-scripts-and-debugging
+set -o errexit   # abort on nonzero exitstatus
+set -o nounset   # abort on unbound variable
+set -o pipefail  # don't hide errors within pipes
```

Comment:
Abort script on error, as [here](https://bertvv.github.io/cheat-sheets/Bash.html#writing-robust-scripts-and-debugging), so that any setup errors are clearer to the user